src/interactive/plots.ts

```diff
@@ -42,7 +42,17 @@ export function showPlotPane() {
     const plotTitle = g_plots.length > 0 ? `Julia Plots (${g_currentPlotIndex + 1}/${g_plots.length})` : 'Julia Plots (0/0)'
     if (!g_plotPanel) {
         // Otherwise, create a new panel
-        g_plotPanel = vscode.window.createWebviewPanel('jlplotpane', plotTitle, { preserveFocus: true, viewColumn: vscode.ViewColumn.Active }, { enableScripts: true })
+    g_plotPanel = vscode.window.createWebviewPanel(
+        'jlplotpane',
+        plotTitle,
+        {
+            preserveFocus: true,
+            viewColumn: vscode.ViewColumn.Beside
```

Comment:
I think that is what we had in a previous iteration and then people didn't like that either and we changed it to the current setting :)

I think maybe we should do the same thing here that I did for the doc browser [here](https://github.com/julia-vscode/julia-vscode/pull/1458/files#diff-95d0de50f6e5fa04a90b3e4cd8e82cc8R81), i.e. remember the last location?

Reply:
Done.